decidim-proposals/app/commands/decidim/proposals/show_endorsement_identities.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    # A command with all the business logic for when a user requests its endorsement related identities.
+    class ShowEndorsementIdentities < Rectify::Command
```

Comment:
Wouldn't it be easier to just use a Rectify::Query here instead of a command?

Reply:
@oriolgual There are 2 queries inside the Command, so you mean to split this Command in 2 Query and invoke those queries directly in the Controller, ok?